Loop/View Controllers/SettingsTableViewController.swift

```diff
@@ -194,6 +198,7 @@ final class SettingsTableViewController: UITableViewController {
                 }
                 cell.textLabel?.text = cgmManager.localizedTitle
                 cell.detailTextLabel?.text = nil
+                cell.accessoryType = .disclosureIndicator
```

Comment:
[nit] curious a disclosure indicator is added here, but not for the pump settings.

Reply:
Oversight on my part.  Good catch.